proms-resources/frontend/src/main/frontend/src/proms/Clinics.json

```
@@ -0,0 +1,8 @@
+{
+  "clinicName": "string",
+  "displayName": "string",
+  "sidebarLabel": "string",
+  "surveyID": "string",
```

Rename to `surveyId`, otherwise the automatically generated label will be `Survey i d`
```suggestion
  "surveyId": "string",
```